vms/platformvm/validators_set_property_test.go

```diff
@@ -0,0 +1,875 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
```

Comment:
nit: I'd rename `validator_set_property_test.go` for consistency with other usage of "validator set" (as opposed to "validators set")

Reply:
done